runtime/devnet/src/config/ismp.rs

```diff
@@ -0,0 +1,58 @@
+use frame_support::traits::Get;
+use frame_system::EnsureRoot;
+use ismp::{error::Error, host::StateMachine, module::IsmpModule, router::IsmpRouter};
+use ismp_parachain::ParachainConsensusClient;
+use sp_std::prelude::*;
+
+use crate::{
+	AccountId, Balance, Balances, Ismp, IsmpParachain, ParachainInfo, Runtime, RuntimeEvent,
+	Timestamp,
+};
+
+impl pallet_ismp::Config for Runtime {
+	type AdminOrigin = EnsureRoot<AccountId>;
+	type Balance = Balance;
+	type ConsensusClients = (ParachainConsensusClient<Runtime, IsmpParachain>,);
+	type Coprocessor = Coprocessor;
+	type Currency = Balances;
+	type HostStateMachine = HostStateMachine;
+	type Mmr = pallet_ismp::NoOpMmrTree<Self>;
+	type Router = Router;
+	type RuntimeEvent = RuntimeEvent;
+	type TimestampProvider = Timestamp;
+	type WeightProvider = ();
+}
+
+impl pallet_ismp_demo::Config for Runtime {
```

Comment:
I'm not sure we should be merging this to main, even if only in a devnet runtime. Perhaps we can instead split out the demo component out into a separate branch/PR that we can easily rebase as/when needed to test ismp?

I'd much rather see our implementation instead of the demo one.